docs/sources/operations/caching.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Caching 
+menuTitle: Caching 
+description: Enable and configure memcached for caching. 
+weight: 100
+keywords:
+  - memcached
+  - caching
+---
+<!-- For more information about how to populate front matter, see [Topic front matter]({{< relref "../../front-matter/" >}}). -->
+
+# Caching
+
+Loki supports caching of index writes and lookups, chunks and query results to
+speed up query performance. This sections describes the recommended Memcached
+configuration to enable caching.
+
+## Before you begin
+
+- It is recommended to deploy four dedicated Memcached clusters.
+- As of 2023-02-01 the `memcached:1.6.17-alpine` is recommended.
+- Consult the Loki ksonnet [memcached](https://github.com/grafana/loki/blob/main/production/ksonnet/loki/memcached.libsonnet) deployment and the ksonnet [memcached library](https://github.com/grafana/jsonnet-libs/tree/master/memcached).
+
+## Steps
+
+To enable and configure Memcached:
+
+1. Deploy each Memcached service with at least three replicas and configure
+   each as follows:
+    1. Chunk cache `--memory-limit=4096 --max-item-size=2m --conn-limit=1024`
+    1. Index lookup cache `--memory-limit=1024 --max-item-size=5m --conn-limit=1024`
+    1. Index write cache `--memory-limit=1024 --max-item-size=1m --conn-limit=1024`
+    1. Query result cache `--memory-limit=1024 --max-item-size=5m --conn-limit=1024`
+
+1. Configure Loki to use the cache.
+    1. If the Helm chart is used
+        1. set `memcached.chunk_cache.host` to the Memecache address of for the chunk cache.
```

Comment:
we also need to set `memcached.chunk_cache.enabled` and `memcached.results_cache.enabled` both to `true.